segmenters/image/FaceNetSegmenter/Dockerfile

```diff
@@ -5,6 +5,8 @@ COPY . /workspace
 WORKDIR /workspace
 
 # Install dependencies
+RUN apt-get update
+RUN apt-get install ffmpeg libsm6 libxext6  -y
```

Comment:
why are these additions needed? if needed, merge `apt-get update` and the rest in one line

Reply:
this is a dependency of opencv-python without which there will be an error

Reply:
so put on the same line to avoid adding an extra Docker layer